rules/cattle-flu.smk

```diff
@@ -163,7 +187,7 @@ rule colors_genome:
     params:
         duplications = "division=division_metadata",
     wildcard_constraints:
-        subtype="h5n1-cattle-outbreak",
+        subtype='h5n1-cattle-outbreak',
```

Comment:
Nice work!  Just pattern-matching -- was this supposed to have an added `|h5n1-d1.1` like the other lines above?

Reply:
And it's back - if interested, the color rule here is to guarantee that the colors used for `division` and `division_metadata` are the same to improve comparison. Important here because 2/3rds of the data is missing 😦